src/Pos/Communication/Server.hs

```diff
@@ -31,13 +31,13 @@ import           Pos.WorkMode                      (WorkMode)
 -- | All listeners running on one node.
 allListeners
     :: (SscListenersClass ssc, WorkMode ssc m)
-    => ([ListenerSpec m], OutSpecs)
-allListeners = mconcatPair
-        [ modifier "block"      $ blockListeners
-        , modifier "ssc"        $ untag sscListeners
-        , modifier "tx"         $ txListeners
-        , modifier "delegation" $ delegationListeners
-        , modifier "update"     $ usListeners
+    => m ([ListenerSpec m], OutSpecs)
+allListeners = mconcatPair <$> sequence
```

Comment:
I mean that `m ([ListenerSpec m], OutSpecs)` and not `([ListenerSpec m], OutSpecs)`